lib/plugin.lazy.js

```diff
@@ -79,21 +80,12 @@ async function loadSentry(ctx, inject) {
   delayedCalls.forEach(([methodName, args]) => Sentry[methodName].apply(Sentry, args))
   <% } %>
 
-  // Inject Sentry to the context as $sentry
-  sentryReadyResolve(Sentry)
   inject('sentry', Sentry)
-  inject('sentryReady', () => Promise.resolve(Sentry))
-  ctx.$sentry = Sentry
-  ctx.$sentryReady = () => Promise.resolve(Sentry)
-
-  <% if (options.lazy.injectLoadHook) { %>
-  // Set noop for sentryLoad so we dont
-  // try to load Sentry more then once
-  const noop = _ => _
-  inject('sentryLoad', noop)
-  ctx.$sentryLoad = noop
-  <% } %>
 
+  sentryReadyResolve(Sentry)
+  sentryReadyPromise = Promise.resolve(Sentry)
```

Comment:
I'm not sure I understand why it's necessary to reassign this variable with a new Promise.

Reply:
Yeah, me neither. But I have seen cases where an await failed because there was no Thenable but just some var. So better save then sry.

Reply:
Still don't understand. `sentryReadyPromise` is a promise from the beginning. It's just reassigned to another promise here.

Reply:
Ah ok, now I understand what you are getting at. It probably doesnt matter much indeed as the promise is already resolved, but I like the explicitness of just overwriting it. If you want I can remove it

Reply:
I'd say that it's enough to just call `sentryReadyResolve` to resolve original promise.